packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/ripple/common";
+@import "@material/ripple/mixins";
+
+.mdc-chip {
+  @include mdc-ripple-surface;
+  @include mdc-ripple-radius;
+  @include mdc-states;
+
+  display: inline-block;
+  border-radius: 16px;
+  background: rgba(0, 0, 0, 0.08);
+  color: rgba(0, 0, 0, 0.87);
+  padding: 7px 12px;
+  margin: 5px;
+  cursor: pointer;
+  overflow: hidden;
+}
+
+.mdc-chip__text {
+  display: inline-block;
+  vertical-align: middle;
```

Comment:
FYI I noticed something funny with this style... with it, chips are 33px tall, but without it, they're 32px tall. (Only checked in Chrome so far.)

Not sure if we're aiming for a specific height for chips, or if this is moot pending the addition of icons anyway.

Reply:
Interesting... the `mdc-chip__text` div is 18px but for some reason the `mdc-chip` content becomes 19px. 

From the `experimental/chip` branch it seems that both `mdc-chip` content and `mdc-chip__icon` height are 24px once icons are added (which is the expected result).

Not sure what's causing this, but a fix for now would be to make the `mdc-chip__text` content 17px since that won't affect anything that comes afterwards. Any thoughts?

Reply:
My first question would be... are chips w/o icons expected to be less tall than chips w/ icons? This would be weird if there's any situation in which you can have a mix of some with icons and some without within the same set, but OTOH if they're all expected to eventually match the with-icons size then this point may become moot.

Reply:
Chips w/o icons are expected to be the same height as chips with icons.
But chips with icons are also 32px, which means that we need to fix this for chips w/o icons anyways.